smart-contracts/witnet-randomness-oracle/randomness-requests.md

```diff
@@ -0,0 +1,46 @@
+# Low-level Randomness Requests
+
+The straight way to fetch randomness from the Witnet oracle is by posting a low-level Randomness Data Request to the [Witnet Request Board](../apis-and-http-get-post-oracle/witnet-request-board.md).
+
+However, you will then need to add logic within your smart contract as to await for an answer from the Witnet side-chain and handle corner-case revert situations at the Witnet level, before feeding your favourite _Pseudo Random Number Generator_ (PRNG) algorithm with the 32-byte random seed provided by the Witnet oracle.
```

Comment:
I find this sentence hard to read. Could you split it in different sentences and remove unnecessary words? Consider the following a quick suggestion
```suggestion
However, using a low-level randomness request has some difficulties. You have to implement it yourself, wait for the answer from the Witnet network and handle corner-case revert situations from Witnet. After that, you will be able to feed your favorite _Pseudo Random Number Generator_ (PRNG) algorithm with the 32-byte random seed provided by the Witnet oracle.
```